csm/conf/configure.py

```diff
@@ -487,3 +489,14 @@ def _set_user_collection(self):
                 record['config']['openldap']['collection'] = const.CORTXUSERS_DN.format(csm_schema_version,base_dn)
         Conf.set(const.DATABASE_INDEX,"models",models_list)
         Conf.save(const.DATABASE_INDEX)
+
+    def _create_perf_stat_topic(self):
+        mb_admin = MessageBusAdmin(admin_id = Conf.get(const.CSM_GLOBAL_INDEX,const.MSG_BUS_ADMIN_ID))
+        message_type = Conf.get(const.CSM_GLOBAL_INDEX,const.MSG_BUS_PERF_STAT_MSG_TYPE)
+        partitions = Conf.get(const.CSM_GLOBAL_INDEX,const.MSG_BUS_PERF_STAT_PARTITIONS)
+        retention_period = Conf.get(const.CSM_GLOBAL_INDEX,const.MSG_BUS_PERF_STAT_RETENTION_PERIOD)
```

Comment:
Change retention_period to retention_size

Reply:
Modifications done